### ISSUE TITLE
Hides the charset and language buttons from the status bar

### ISSUE THREAD

**Issue by CyrilMazur** [nesQuick/atom-wordcount#28]
Is it a feature actually? It would make sense to hide the language (since words count is rather for text files) but the charset is important.

**Comment by nesQuick** [comment]
I'm afraid I do not really understand your feature request. But I'm happy to take a look at a pull request so we can talk code :)

**Comment by CyrilMazur** [comment]
Sorry, better with a screenshot. Before I installed the plugin, it used to show «UTF-8 Plain text» under the red line.
![screen shot 2015-03-29 at 22 50 34](https://cloud.githubusercontent.com/assets/464216/6886134/756ee1bc-d666-11e4-9f96-63d4b7c3fd5a.png)

**Comment by nesQuick** [comment]
It works for me, but if you can find the issue, I'm happy to accept pull requests

![screen_shot_2015-03-29_at_23_34_42](https://cloud.githubusercontent.com/assets/584259/6887861/59285dd4-d66c-11e4-97b4-fd8735229a1e.png)

**Comment by nesQuick** [comment]
You might want to ensure that you are using the latest version.

**Comment by CyrilMazur** [comment]
I'm using 2.1.0. From developer tools, I found out that if I remove the style="display:block;" attribute from the .word-count inline-block div (which displays then as inline-block, hence the name), the encoding and grammar selectors come back. So the problem is this "display:block" that's not supposed to be there.

I have never written in coffee script before, but I'll try to hunt down the bug and submit a PR if I find it.

![screen shot 2015-03-31 at 17 04 35](https://cloud.githubusercontent.com/assets/464216/6915614/244ee454-d7c8-11e4-8e13-d8ba5872fa9b.png)

**Comment by nesQuick** [comment]
:+1: awesome let me know if you need any help. Atom has great docs as well [atom.io](http://atom.io)

**Comment by CyrilMazur** [comment]
Maybe this line actually https://github.com/nesQuick/atom-wordcount/blob/master/lib/main.coffee#L31 If that is the jquery show() method, then I recommend the following instead:

```javascript
if current_file_extension in extensions
  view.css("display", "inherit"); // inherit from the inline-block class
else
  view.css("display", "none"); // force display none
```

**Comment by nesQuick** [comment]
I don't know what atom is doing internaly, but it's open source so you can investigate here https://github.com/atom/atom

**Comment by CyrilMazur** [comment]
Here's the PR https://github.com/nesQuick/atom-wordcount/pull/29

**Comment by nesQuick** [comment]
Thank you very much @CyrilMazur for your contribution! :heart: I just published this version as `2.2.0` :+1:

**Comment by CyrilMazur** [comment]
Happy to help :)
